platform/mbed_lib.json

```diff
@@ -32,6 +32,10 @@
         },
         "EFR32": {
             "stdio-baud-rate": 115200
+        },
+        "WUNDERBAR": {
+            "stdio-baud-rate": 115200,
+            "default-serial-baud-rate": 115200
```

Comment:
Where is this coming from? Is it that any other than stdio is set by default to 115k? 

stdio baud rate - 115k, is it as the interface defautls to 115k?

Reply:
Wunderbar uses UART to communicate with WiFi modem which has default 115k baud rate. If this is not set, mbed_retarget tries to set baud rate to 9600 and this confuses WiFi modem.